web/looking-good/src/pages/user-profile-edit.js

```diff
@@ -46,117 +64,242 @@ const UserProfileEdit = () => {
+                  <h2>Now for the fun part....</h2>
+                  <p>
+                    Record a simple video to help your clients maintain their look until they can
+                    see you again in person. Consider showing ways to style instead of cutting your
+                    own bangs, giving advice for those of us struggling with our natural color,
+                    or telling us what to do now that our gel manis have reached an embarrassing
+                    grow-out length. Or just stay in touch and show us what you’ve been up to!
+                  </p>
+                  <FormControl className={classes.formInputs}>
+                    <TextField
+                      aria-describedby={'video_url'}
+                      error={!!formProps.errors.video_url}
+                      helperText={'Facebook, YouTube or Instagram video'}
+                      id={'video_url'}
+                      label={'Link to your video'}
+                      name={'video_url'}
+                      onBlur={formProps.handleBlur}
+                      onChange={formProps.handleChange}
+                      value={formProps.values.video_url}
+                    />
+                  </FormControl>
+                  <p>
+                    Give us a few words to preview what we’ll see in your vid and encourage your
+                    clients to share, share, share! Make sure to remind them to tag you and
+                    use <strong>#LookingGoodCbus</strong> when they post about how awesome you are!
+                  </p>
+                  <FormControl className={classes.formInputs}>
+                    <TextField
+                      aria-describedby={'blurb'}
+                      error={!!formProps.errors.blurb}
+                      helperText={'A few words to preview what we\'ll see in your vid'}
+                      id={'blurb'}
+                      label={'Video Description'}
+                      multiline={true}
+                      name={'blurb'}
+                      onBlur={formProps.handleBlur}
+                      onChange={formProps.handleChange}
+                      value={formProps.values.blurb}
+                    />
+                  </FormControl>
+
+                  <FormErrors errors={formErrors} />
+
+                  <Button
+                    color={'primary'}
+                    disabled={formProps.isSubmitting}
+                    onClick={formProps.handleSubmit}
+                    size={'large'}
+                    style={{ align: 'right' }}
+                    type={'submit'}
+                    variant={'contained'}
+                  >
+                    Save Changes
+                  </Button>
                 </Grid>
-              </CardActions>
+              </Grid>
             </div>
-          )}
-        </Formik>
-
-      </Card>
+          </div>
+        )}
+      </Formik>
     </Container>
   )
 }
 
+// noinspection JSUnusedGlobalSymbols
```

Comment:
What is this line doing?

Reply:
That's a Rubymine instruction for the IDE linter, removed.